devtools/server/actors/replay/connection.js

```diff
@@ -462,7 +474,7 @@ class Recording extends EventEmitter {
   _onNewSourcemap(params) {
     this._lockRecording(params.recordingId);
 
-    this._resourceUploads.push(uploadAllSourcemapAssets(params).catch(err => {
+    this._resourceUploads.push(uploadAllSourcemapAssets(params, this.browser).catch(err => {
```

Comment:
I think we should probably pass `this.browser.contentPrincipal` here, otherwise if the page were to navigate while we were still fetching sourcemaps for the previous page, we could end up using the wrong principal.